src/test/java/com/redhat/che/workspace/services/telemetry/woopra/AnalyticsManagerTest.java

```diff
@@ -0,0 +1,48 @@
+package com.redhat.che.workspace.services.telemetry.woopra;
+
+import com.redhat.che.workspace.services.telemetry.woopra.exception.WoopraCredentialException;
+import io.quarkus.test.Mock;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class AnalyticsManagerTest {
+
+    @Mock
+    AnalyticsProvider analyticsProvider;
+
+    @Mock
+    HttpUrlConnectionProvider httpUrlConnectionProvider;
+
+    @Test
+    public void willNotStartWithoutWriteKeyEndpoint() {
+        assertThrows(WoopraCredentialException.class, () -> {
+            AnalyticsManager am = new AnalyticsManager(null,
+                    null,
+                    null,
+                    null,
+                    "https://fake.che.openshift.io/api",
```

Comment:
I khow this is only test, but let's avoid Hoste Che specific constants in the upstream code